src/middlewares/error-sender.js

```diff
@@ -3,7 +3,8 @@ const getErrorCode = error => error.status || error.code || 500;
 const getCommonErrorResponseObject = error => ({
   name: error.name,
   message: `${error.name}(${getErrorCode(error)}): ${error.message}`,
-  code: getErrorCode(error)
+  code: getErrorCode(error),
+  payload: error.payload
```

Comment:
Add it for additional information like a unique field what was duplicate when create users.